src/wp-includes/script-loader.php

```diff
@@ -2209,6 +2209,7 @@ function _print_scripts() {
 			echo "\n<script{$type_attr}>\n";
 			echo "/* <![CDATA[ */\n"; // Not needed in HTML 5.
 			echo $wp_scripts->print_code;
+			echo sprintf( "\n//# sourceURL=%s\n", rawurlencode( 'js-inline-concat-' . $concat ) );
```

Comment:
There's example output in tests, but this outputs a sourceURL like this:
```js
//# sourceURL=js-inline-concat-one%2Ctwo
```

`$concat` is a list of the concatenated script names, `one,two` in this case. It should work well as a descriptive identifier. For example, on my test site on the `wp-admin/customizer.php` page I get an inline script that's identified in the Chrome devtools source panel as `js-inline-concat-heartbeat,customize-widgets,react,react-dom,react-jsx-runtime,wp-polyfill,wp-url`.

Reply:
I like this, even if it is short-lived. 👍🏻